simapp/simd/cmd/root.go

```diff
@@ -173,7 +180,7 @@ func newApp(logger log.Logger, db dbm.DB, traceStore io.Writer, appOpts serverty
 		logger, db, traceStore, true, skipUpgradeHeights,
 		cast.ToString(appOpts.Get(flags.FlagHome)),
 		cast.ToUint(appOpts.Get(server.FlagInvCheckPeriod)),
-		encodingConfig,
+		simapp.MakeEncodingConfig(), // Ideally, we would reuse the one created by NewRootCmd.
```

Comment:
This was originally assigned as a private variable in this package. Why was it undone?